src/extension.ts

```diff
@@ -25,11 +25,11 @@ export async function activate(context: vscode.ExtensionContext) {
     await callWithTelemetryAndErrorHandling('azurePipelines.activate', async (activateContext: IActionContext) => {
         activateContext.telemetry.properties.isActivationEvent = 'true';
         telemetryHelper.initialize(activateContext, 'activate');
-        telemetryHelper.setTelemetry('configurePipelineEnabled', `${configurePipelineEnabled}`);
         await telemetryHelper.executeFunctionWithTimeTelemetry(
             async () => {
                 await activateYmlContributor(context);
                 if (configurePipelineEnabled) {
+                    telemetryHelper.setTelemetry('configurePipelineEnabled', `${configurePipelineEnabled}`);
```

Comment:
the value of const `configurePipelineEnabled` is true or false, we can always log this irrespective of it being enabled or not. 
Our query will be like properties.configurePipelineEnabled == 'true'. 

We can revert this change

Reply:
Got it. Reverted.